src/main/java/ServiceLayer/Response.java

```diff
@@ -1,39 +1,33 @@
 package ServiceLayer;
+
```

Comment:
pls add the main reason to use this class

Reply:
Again, I think this is clear from the name